homeassistant/components/hassio/services.yaml

```diff
@@ -73,6 +73,23 @@ host_update:
       description: Optional or it will be use the latest version.
       example: "0.3"
 
+snapshot_full:
+  description: Create a full snapshot.
+  fields:
+    name:
```

Comment:
There's a password parameter too:
https://github.com/home-assistant/core/blob/c9af5287bdc6767fa219fb8c295d4d0385ff1c12/homeassistant/components/hassio/__init__.py#L77-L79

Reply:
I see that, partial also has that param and name too.  I'll update it.